views/guide.lavender

```diff
@@ -392,7 +394,8 @@ block content
         pre
           code.language-jade.
             p
-            a(href=&#39;/some/place&#39;)- click here for some cool stuff.
+              a(href=&#39;/some/place&#39;)- click here for some cool stuff.
+              | |.
```

Comment:
yea it just looks weird because we're rendering plaintext lavender with lavender